ScoutSuite/output/data/html/partials/details_for_project.html

```diff
@@ -0,0 +1,17 @@
+
+  <!-- Generic partial for project-then-resource -->
```

Comment:
What does _project-then-resource_ mean?

Reply:
It's a formulation carried over from AWS' `details_for_region.html` referring to the UI showing a project (or region for AWS) followed by the resource.

Reply:
I mean can we put a more meaningful comment, haha. It's not a great comment, you have to be pretty familiar with the product to understand it IMO.

Reply:
Put a more meaningful one!